webapp/src/components/viewHeader/viewHeaderPropertiesMenu.test.tsx

```diff
@@ -0,0 +1,67 @@
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+import React, {ReactElement} from 'react'
+import {render, screen} from '@testing-library/react'
+import {Provider as ReduxProvider} from 'react-redux'
+import configureStore from 'redux-mock-store'
+
+import '@testing-library/jest-dom'
+import userEvent from '@testing-library/user-event'
+
+import {IntlProvider} from 'react-intl'
+
+import {TestBlockFactory} from '../../test/testBlockFactory'
+
+import ViewHeaderPropertiesMenu from './viewHeaderPropertiesMenu'
+
+const wrapIntl = (children: ReactElement) => (
+    <IntlProvider locale='en'>{children}</IntlProvider>
+)
```

Comment:
This function can be imported from [testUtils.tsx](https://github.com/mattermost/focalboard/blob/main/webapp/src/testUtils.tsx#L6)

Reply:
Done ;)